core/domain/draft_upgrade_services.py

```diff
@@ -312,6 +312,23 @@ def _convert_html_in_draft_change_list(
                 })
         return draft_change_list
 
+    @classmethod
+    def _convert_states_v52_dict_to_v53_dict(
+        cls, draft_change_list: List[exp_domain.ExplorationChange]
+    ) -> List[exp_domain.ExplorationChange]:
+        """Converts draft change list from state version 52 to 53. State
+        version 52 adds catchMisspellings customization_arg to
+        TextInput interaction.
```

Comment:
Please explain why there won't be any changes to the draft.

Reply:
Fixed.